### DISCUSSION TITLE
How to put date in the log pattern QUILL_STRING?

### DISCUSSION THREAD

**Discussion by xanderdunn** [odygrd/quill#106] [Q&A] [answered]
A typical log pattern `QUILL_STRING` looks like: `%(ascii_time) [%(thread)] %(filename):%(lineno) LOG_%(level_name)%(message)`. `ascii_time` puts the time in the format `10:26:55.738221048`, but how do I include the date?

Are the accepted patterns documented somewhere? I wasn't able to find a list of them.

`%(date)` wasn't recognized

**Comment by bl-ue** [comment, marked as answer]
I think you might be looking for this: https://github.com/odygrd/quill/wiki/4.-Formatters#customising-the-timestamp

**Reply by xanderdunn** [reply]
That's it, thank you!  `%Y-%m-%d %H:%M:%S.%Qms %z`
